cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -73,11 +73,19 @@ func (p *plugin) getArgs() []string {
 		command = p.Command
 	}
 
+	args = append(args, command)
+
+	// Include step name as the CI job name for EngFlow
+	drone_step_name := os.Getenv("DRONE_STEP_NAME")
+	if drone_step_name != "" {
+		args = append(args, "--bes_keywords=engflow:CiCdJobName="+drone_step_name)
```

Comment:
After a read through https://docs.engflow.com/web_ui/index.html#ci-system, it seems like there are a number of other flags that might make sense to set here as well. Is that not necessary, or these are just set elsewhere?

Reply:
They're set in an initialization step I linked to in the description, but in case you don't have access to the repo it's this

```bash
echo "build --bes_keywords=engflow:CiCdPipelineName=$DRONE_STAGE_NAME"
# echo "build --bes_keywords=engflow:CiCdJobName=$DRONE_STEP_NAME"  https://github.com/kanopy-platform/drone-bazelisk-ecr/pull/13
echo "build --bes_keywords=engflow:CiCdUri=$DRONE_BUILD_LINK"
echo "build --bes_keywords=engflow:BuildScmRemote=$DRONE_REPO_LINK"
echo "build --bes_keywords=engflow:BuildScmBranch=$DRONE_COMMIT_BRANCH"
echo "build --bes_keywords=engflow:BuildScmRevision=$DRONE_COMMIT"
```

>I'm less sure if that's a good idea because it's pretty easy to do that part in a shared pipeline step. The thing that makes CiCdJobName different is that it is different for every step.

If you think we can put them all here then I'll move them. There's a couple more related to credentials that are not drone- or engflow-standard, I just picked environment variable names and drone secret names for myself in 10gen/mms. So I'm even less sure if those should be incorporated here. But the BES keywords definitely could be.

Reply:
Yeah, those all seem pretty standard, so it probably makes sense to just set as well. More reason to probably gate these behind a `settings` flag that's off by default though.